### ISSUE TITLE
Does not work on RPi with Raspbian 9 Strech

### ISSUE THREAD

**Issue by GsakuL** [git-developer/ti-cc-tool#3]
I'm currently trying to remotely update a Sonoff Coordinator.
Sadly this container does not work at all.
`Docker version 19.03.15, build 99e3ed8919`
```
$ ls
CC1352P2_CC2652P_launchpad_coordinator_20221226.hex  CC1352P2_CC2652P_launchpad_coordinator_20221226.zip  update.sh
```
`update.sh`:
```bash
docker run \
        --rm \
        --device /dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20221201142743-if00:/dev/ttyUSB0 \
        -e FIRMWARE_URL="https://github.com/Koenkk/Z-Stack-firmware/raw/master/coordinator/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_coordinator_20221226.zip" \
        ckware/ti-cc-tool -ewv -p /dev/ttyUSB0 --bootloader-sonoff-usb
```
When I try to run it (as sudo) it just fails with an unhelpful message
```
Downloading firmware from https://github.com/Koenkk/Z-Stack-firmware/raw/master/coordinator/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_coordinator_20221226.zip
ERROR: Download of firmware was not successful
```
also I don't understand how the "FIRMWARE_FILE" EnvVar is supposed to work. It just says
```
ERROR: Unreadable firmware file 'CC1352P2_CC2652P_launchpad_coordinator_20221226.zip'
```

**Comment by git-developer** [comment]
What version of `libseccomp2` is installed on the host system (run `dpkg -l libseccomp2` to check)? The container os [requires](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.13.0#time64_requirements) 2.4.2 or newer. If your version is lower, please check if an update is possible and report back.

**Comment by git-developer** [comment]
In case it's not feasible to upgrade `libseccomp2`, you can try the legacy image that I just built. Replace `ckware/ti-cc-tool` with `ckware/ti-cc-tool:legacy-latest` to do so.

Please consider upgrading your host OS. Stretch was released in 2017, latest update in 2020, no LTS security updates since 06/2022.

**Comment by GsakuL** [comment]
I'm sorry for all that trouble.
I had no idea that a host dependency could cause this. 
Indeed `libseccomp2` is `2.3.1`.
Since you already made an effort, I tried the legacy image and it worked, but the stick seems unresponsive.
I actually tried and set up `cc2538-bsl` to double check. I've wanted to use a venv, to not "cause further harm", but it was an even greater hassle than it should have been, due to some unexplained glitches. (which is why it took so long to reply) But at last: the same error.
So I'll be heading there tomorrow to have a look myself. I suspect a SD failure, and/or a not properly plugged in stick.
And yes, you're right. Stretch is old -I'll have a word with the owners.

I'll close this issue, since it's a user-error.
Again, very sorry for the trouble
